server/models/meteorological.py

```diff
@@ -1,18 +1,20 @@
-from sqlalchemy import Column, Float, Integer, String
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
 
 from server.db.base_class import Base
 
 
-class Historical_Meteorological(Base):
-    __tablename__ = "Historical Meteorological Data"
+class Meteorological(Base):
```

Comment:
Import this class in relative [`__init__.py` file](https://github.com/clairBuoyant/server/blob/main/server/models/__init__.py) for convenience.

And also in [this file](https://github.com/clairBuoyant/server/blob/main/server/db/base.py) to support working with `alembic` for migrations.